src/resources/scepter.ts

```diff
@@ -0,0 +1,188 @@
+    value: () => 8 * get("valueOfAdventure"),
+    type: "special",
+  },
+  {
+    skill: $skill`Aug. 18th: Serendipity Day!`,
+    value: () => 1000, // Dummy value; we should some day calculate this based on free fight count, careful to avoid circular imports
+    type: "buff",
+  },
+  {
+    skill: $skill`Aug. 24th: Waffle Day!`,
+    value: () => 3 * garboValue($item`waffle`),
+    type: "summon",
+  },
+  {
+    skill: $skill`Aug. 25th: Banana Split Day!`,
+    value: () => garboValue($item`banana split`),
+    type: "summon",
+  },
+  {
+    skill: $skill`Aug. 26th: Toilet Paper Day!`,
+    value: () => garboValue($item`handful of toilet paper`),
+    type: "summon",
+  },
+  {
+    skill: $skill`Aug. 29th: More Herbs, Less Salt  Day!`,
+    value: () => 3 * garboValue($item`Mrs. Rush`),
+    type: "summon",
+  },
+  {
+    skill: $skill`Aug. 30th: Beach Day!`,
+    value: () =>
+      100 +
+      (globalOptions.ascend
+        ? 0
+        : clamp(
+            7 -
+              getModifier(
+                "Adventures",
+                Item.all()
+                  .filter((i) => have(i) && toSlot(i) === $slot`acc1` && canEquip(i))
+                  .sort((a, b) => getModifier("Adventures", b) - getModifier("Adventures", a))[2] ??
+                  $item.none,
+              ),
+            0,
+            7,
+          ) * get("valueOfAdventure")),
+    type: "summon",
+  },
+  {
+    skill: $skill`Aug. 31st: Cabernet Sauvignon  Day!`,
+    value: () => 2 * garboValue($item`bottle of Cabernet Sauvignon`),
+    type: "summon",
+  },
+];
+
+let bestScepterSkills: ScepterSkill[] | null = null;
+function getBestScepterSkills(): ScepterSkill[] {
+  return (bestScepterSkills ??= SKILL_OPTIONS.filter(
+    ({ skill }) => AugustScepter.todaysSkill() !== skill,
+  )
+    .sort((a, b) => b.value() - a.value())
+    .splice(0, clamp(5 - get("_augSkillsCast"), 0, 5)));
+}
+
+export function shouldAugustCast(skill: Skill) {
+  return (
+    AugustScepter.have() &&
+    ((getBestScepterSkills().some((s) => skill === s.skill) &&
+      skill.dailylimit &&
+      get("_augSkillsCast") < 5) ||
+      (AugustScepter.todaysSkill() === skill && !AugustScepter.getTodayCast()))
+  );
+}
+
+function summonTask({ skill }: ScepterSkill) {
+  return {
+    name: skill.name,
+    completed: () => !shouldAugustCast(skill),
+    do: () => useSkill(skill),
+  };
+}
+
+export function augustSummonTasks() {
+  return AugustScepter.have()
+    ? SKILL_OPTIONS.filter(({ type }) => type === "summon").map(summonTask)
+    : [];
+}
+
+export function castAugustScepterBuffs() {
+  if (AugustScepter.have()) {
+    for (const { skill } of SKILL_OPTIONS.filter(
+      ({ skill, type }) => shouldAugustCast(skill) && type === "buff",
+    )) {
+      useSkill(skill);
+    }
+
+    const today = SKILL_OPTIONS.find(({ skill }) => skill === AugustScepter.todaysSkill());
+    if (today && !AugustScepter.getTodayCast()) useSkill(today.skill);
```

Comment:
are we sure garbo wants to do the cat fight? it might actually be a waste for it to do so.